DIY-Peach!.md

```diff
@@ -0,0 +1,19 @@
+Guide for growing a Peach
+====
+
+As inspired by [Patterns for Decentralized Organizing](https://docs.wixstatic.com/ugd/8246de_248a47ddbd3c4d209667ff5508ee300b.pdf), [IDOARRT](https://toolbox.hyperisland.com/idoarrt-meeting-design), [9 Artifacts To Seed a Project Team](http://kfjournal.org/wp/index.php/2015/10/19/9-artifacts-to-seed-a-project-team/) and [Pods](https://handbook.enspiral.com/guides/pods.html) and [Stewardship](https://loomio.coop/stewarding.html) from Enspiral. 
+
+This is a proposal for how you can [grow a peach](https://github.com/scuttlebutt-eu/important-documents/pull/33). A peach is a form of working group which together built up the European Scuttlebutt Consortium tree. Feel free to hack!  
```

Comment:
Before merging this PR, I don't think we should link to other PRs. Documents should link to other documents, and PRs aren't such.